src/improvements/tasks/TaskRunner.sol

```diff
@@ -57,4 +58,21 @@ contract Runner is Script {
         run();
         vm.dumpState(dumpStatePath);
     }
+
+    /// @notice Returns Useful function to tell if a task is nested or not based on the task config
```

Comment:
Added this to Runner.sol and renamed it to TaskRunner.sol (instead of giving it its own file).

Reply:
Can be a separate PR, but we should add contract-level to this contract (and any others that are missing it) to explain its purpose

Reply:
Sounds good I've made a note.